src/history/history-and-comments.component.tsx

```diff
@@ -237,55 +256,68 @@ const HistoryAndComments: React.FC<{
     <div className={styles.historyAndCommentsContainer}>
       {isLoading && <DataTableSkeleton role="progressbar" />}
       {isError && <p>{t("error", "Error")}</p>}
-      {dispenses &&
-        dispenses.map((dispense) => {
-          return (
-            <div key={dispense.id}>
-              <h5
-                style={{
-                  paddingTop: "8px",
-                  paddingBottom: "8px",
-                  fontSize: "0.9rem",
-                }}
-              >
-                {dispense.performer && dispense.performer[0]?.actor?.display}{" "}
-                {generateDispenseVerbiage(dispense)} -{" "}
-                {formatDatetime(parseDate(getDateRecorded(dispense)))}
-              </h5>
-              <Tile className={styles.dispenseTile}>
-                {generateMedicationDispenseActionMenu(dispense)}
-                <MedicationEvent
-                  medicationEvent={dispense}
-                  status={generateDispenseTag(dispense)}
-                />
-              </Tile>
-            </div>
-          );
-        })}
-      {requests &&
-        requests.map((request) => {
-          return (
-            <div key={request.id}>
-              <h5
-                style={{
-                  paddingTop: "8px",
-                  paddingBottom: "8px",
-                  fontSize: "0.9rem",
-                }}
-              >
-                {request.requester.display}{" "}
-                {t("orderedMedication ", "ordered medication")} -{" "}
-                {formatDatetime(prescriptionDate)}
-              </h5>
-              <Tile className={styles.requestTile}>
-                <MedicationEvent
-                  medicationEvent={request}
-                  status={<Tag type="green">{t("ordered", "Ordered")}</Tag>}
-                />
-              </Tile>
-            </div>
-          );
-        })}
+      {medicationRequestBundles &&
```

Comment:
Don't think there's much change here besides the rework to use medicatoinRequestBundles